README.md

```diff
@@ -111,7 +111,7 @@ If you are looking to contribute then explore this list, look at [`first-timers-
 
 - [phpMyAdmin](https://github.com/phpmyadmin/phpmyadmin/labels/newbie) _(label: newbie)_ <br> Admin interface for MySQL written in PHP.
 - [Deployer](https://github.com/deployphp/deployer/labels/good%20for%20beginner) _(label: good for beginner)_ <br> A deployment tool written in PHP with support for popular frameworks out of the box.
-- [Piwik](https://github.com/piwik/piwik/labels/Easy%20pick) _(label: Easy Pick)_ <br> Piwik is the leading Free/Libre open analytics platform.
+- [Matomo](https://github.com/matomo-org/matomo/labels/help%20wanted) _(label: Easy Pick)_ <br> Matomo is the leading Free/Libre open analytics platform.
```

Comment:
Hi @sammysgame 

Could you please update the Label in brackets as well?

Thanks